tests/test_agents.py

```diff
@@ -973,6 +973,30 @@ async def test_clinical_tool_usage(agent_test_settings) -> None:
     ), "No clinical trials were put into contexts"
 
 
+@pytest.mark.asyncio
+async def test_citation_formatting(agent_test_settings):
```

Comment:
What do you think of collapsing this test into a subtest of `test_agent_types`, and it would actually:

- Decrease CI runtime (since we're already running the agent in those tests)
- Increase the coverage of the assertions, since more agents types are used there